internal/handlers/pg/pgdb/settings_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgdb
+
+import (
+	"testing"
+
+	"github.com/jackc/pgx/v4"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/FerretDB/FerretDB/internal/util/testutil"
+)
+
+func TestSettings(t *testing.T) {
+	t.Parallel()
+
+	ctx := testutil.Ctx(t)
+
+	pool := getPool(ctx, t)
+	databaseName := testutil.DatabaseName(t)
+	collectionName := testutil.CollectionName(t)
+	setupDatabase(ctx, t, pool, databaseName)
+
+	err := pool.InTransactionRetry(ctx, func(tx pgx.Tx) error {
+		nameCreated, _, err := ensureSettings(ctx, tx, databaseName, collectionName)
+		if err != nil {
+			return err
+		}
```

Comment:
why can't we require no error here? :thinking: 
```suggestion
		require.NoError(t, err)
```

Reply:
In the case of tests running in parallel, there could be (and there is) a situation when this code returns an error. It's, for example, when to test reach `ensureTest` and attempt to create indexes for the settings table in parallel. This is why we use `InTransactionRetry` here.

Reply:
Got it. Hence we are testing it within `InTransactionRetry` and checking that error instead of this error :pray:

Reply:
added a comment